### PR TITLE
Merge functions, improve seekbar

### PR DESCRIPTION
**Pull request by nyagami**

### CODE REVIEW

File: src/screens/reader/ReaderScreen.js [hunk 1]
```diff
@@ -392,16 +397,7 @@ const ChapterContent = ({ route, navigation }) => {
...
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToPrevChapter
-          }
-          onSwipeLeft={
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToNextChapter
-          }
+          onSwipe={navigateToChapterBySwipe}
```
Comment (rajarsheechatterjee) [inline src/screens/reader/ReaderScreen.js:400]: `swipeGestures && (!useWebViewForChapter || !wvUseNewSwipes)` should be true for Swipe Navigation to be enabled.

Comment (nyagami) [reply src/screens/reader/ReaderScreen.js:400]: Why we cant use `swipeGestures` for `webview`. Also I dont know what `wvUseNewSwipes` does.
So only swipeGestures is enough imo

Comment (nyagami) [reply src/screens/reader/ReaderScreen.js]: So you should be unnecessary right now?. Cause we can swipe in Web mode without it

File: src/screens/reader/ReaderScreen.js [hunk 2]
```diff
@@ -392,16 +397,7 @@ const ChapterContent = ({ route, navigation }) => {
...
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToPrevChapter
-          }
-          onSwipeLeft={
-            swipeGestures &&
-            (!useWebViewForChapter || !wvUseNewSwipes) &&
-            navigateToNextChapter
-          }
+          onSwipe={swipeGestures && !wvUseNewSwipes && navigateToChapterBySwipe}
```
Comment (rajarsheechatterjee) [inline src/screens/reader/ReaderScreen.js:400]: Render HTML and `wvUseNewSwipes` both should be true or only `swipeGestures`.

File: src/screens/reader/ReaderScreen.js (hunk 1 above)
Comment (rajarsheechatterjee) [reply src/screens/reader/ReaderScreen.js]: Then we would also have to remove it from the setting, since we are combining them.

Comment (nyagami) [reply src/screens/reader/ReaderScreen.js]: okay, lemme try

File: src/redux/settings/settings.reducer.js [hunk 3]
```diff
@@ -63,17 +63,14 @@ export const initialState = {
   fullScreenMode: true,
   swipeGestures: false,
   showScrollPercentage: true,
-  useWebViewForChapter: false,
-  wvUseNewSwipes: false,
-  wvShowSwipeMargins: true,
-  wvUseVolumeButtons: false,
+  ShowSwipeMargins: true,
```
Comment (rajarsheechatterjee) [inline src/redux/settings/settings.reducer.js:66]:
```suggestion
  showSwipeMargins: true,
```

**Review by rajarsheechatterjee — APPROVED** [review]

**Pull request merged**